### PR TITLE
[magmad] Add GRPC logging capabilities to magmad

### LINKED ISSUE #4826: [lte][agw] Add GRPC tracing support on AGW

### PR DESCRIPTION
**Pull request by uri200**
Signed-off-by: Oriol Batalla <obatalla@fb.com>

## Summary

Add logging capabilities to magmad 

Enabled by `magmad.yml`

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: orc8r/gateway/python/magma/magmad/rpc_servicer.py [hunk 1]
```diff
@@ -48,7 +51,7 @@ def __init__(self,
                  service_manager: ServiceManager,
                  mconfig_manager: MconfigManager,
                  command_executor: CommandExecutor,
-                 loop):
+                 loop, print_grpc_payload: bool = False):
```
Comment (themarwhal) [inline orc8r/gateway/python/magma/magmad/rpc_servicer.py]: Would be good to insert a new line after `loop,` for readability
(And if you feel like it, do you mind adding type annotation for loop?)

File: orc8r/gateway/python/magma/magmad/rpc_servicer.py [hunk 2]
```diff
@@ -344,3 +354,15 @@ def create_probe_protos(probes):
         ) for param in traceroute_param_protos]
         traceroute_results = traceroute.traceroute(traceroutes_to_exec)
         return map(create_result_proto, traceroute_results)
+
+    def _print_grpc(self, message):
```
Comment (themarwhal) [inline orc8r/gateway/python/magma/magmad/rpc_servicer.py:358]: I think we should pull this function into `magma.common.rpc_utils` maybe? Could be somewhere else, but we seem to have this function in all the services.

File: orc8r/gateway/python/magma/magmad/main.py [hunk 3]
```diff
@@ -218,7 +218,7 @@ async def bootstrap_success_cb(certs_generated: bool):
     magmad_servicer = MagmadRpcServicer(
         service,
         services, service_manager, get_mconfig_manager(), command_executor,
-        service.loop,
+        service.loop, service.config.get('print_grpc_payload', False)
```
Comment (themarwhal) [inline orc8r/gateway/python/magma/magmad/main.py]: Would be good to insert a new line after loop, for readability
(And if you feel like it, do you mind adding type annotation for loop?)

File: orc8r/gateway/python/magma/magmad/rpc_servicer.py (hunk 2 above)
Comment (uri200) [reply orc8r/gateway/python/magma/magmad/rpc_servicer.py:358]: yep we do, didn't know we had `rpc_utils`

Will do a separate PR to migrate this function in all the services we introduced it

**Review by themarwhal — APPROVED** [review]

File: docs/readmes/lte/dev_notes.md [hunk 4]
```diff
@@ -112,7 +112,7 @@ the section previous (`/var/log/syslog`). By default, it is disabled.
 Note GRPC printing is independent of the `log_level`. So you can enabled GRPC
 printing even that your level is set as `INFO`.
 
-To enable GRPC logging for `pipelined`, `mobilityd`, `directoryd` or
+To enable GRPC logging for `magmad`, `pipelined`, `mobilityd`, `directoryd` or
 `subcriberdb` you can modify this line on the `/etc/magma/<service_name>.yml`
```
Comment (ardzoht) [inline docs/readmes/lte/dev_notes.md]: nit: typo in subscriberdb

**Review by ardzoht — APPROVED** [review]
minor comment, overall lgtm

**Review by mpgermano — APPROVED** [review]

**Pull request merged**